src/pages/claim-details/claim-actions.test.tsx

```diff
@@ -211,6 +216,39 @@ describe('<ClaimActions />', () => {
         expect(appealButton).toBeDisabled();
       });
     });
+
+    describe('when the ruling has been appealed', () => {
+      it('shows that it has been appealed to Kleros', () => {
+        claim.dispute = {
+          id: '1',
+          status: 'Waiting',
+          ruling: 'PaySettlement',
+          appealedBy: '0x153EF0B488148k0aB0FED112334',
```

Comment:
Isn't this too short for an address?

Reply:
Yeah but I don't see why that matters?

Reply:
Just caught my eye. I understand it doesn't matter for the tests though. Feel free to leave it as is.